templates/etc/nginx/sites-available/fpm.conf.erb

```diff
@@ -41,7 +41,8 @@ server {
 
   location ~ ^(.+\.php)(.*)$ {
     include fastcgi_params;
-
+    #allows to handle 404 in case of a file not found in fpm
+    try_files $uri =404;
```

Comment:
What this line says is to reply with a 404 if it doesn't find the file with the default nginx error page. 

Currently if it doesn't find the php page it replies with File not found.